mappings/net/minecraft/world/World.mapping

```diff
@@ -1,16 +1,57 @@
 CLASS net/minecraft/class_18 net/minecraft/world/World
+	FIELD field_179 lightingQueueCount I
 	FIELD field_180 isRemote Z
+	FIELD field_181 lightingQueue Ljava/util/List;
+	FIELD field_182 entitiesToUnload Ljava/util/List;
+	FIELD field_183 scheduledUpdates Ljava/util/TreeSet;
+	FIELD field_184 scheduledUpdateSet Ljava/util/Set;
+	FIELD field_185 blockEntityUpdateQueue Ljava/util/List;
+	FIELD field_186 worldTimeMask J
+	FIELD field_187 timeLoaded J
+	FIELD field_188 allPlayersSleeping Z
+	FIELD field_189 tempCollisionBoxes Ljava/util/ArrayList;
+	FIELD field_190 processingDeferred Z
+	FIELD field_191 lightingUpdateCount I
+	FIELD field_192 allowMonsterSpawning Z
+	FIELD field_193 allowMobSpawning Z
+	FIELD field_194 activeChunks Ljava/util/Set;
+	FIELD field_195 ambientSoundCounter I
+	FIELD field_196 tempEntityList Ljava/util/List;
+	FIELD field_197 instantTick Z
+	FIELD field_198 entities Ljava/util/List;
+	FIELD field_199 blockEntities Ljava/util/List;
+	FIELD field_200 players Ljava/util/List;
+	FIELD field_201 globalEntities Ljava/util/List;
+	FIELD field_202 ambientDarkness I
+	FIELD field_203 lcgBlockSeed I
+	FIELD field_204 lcgBlockSeedIncrement I
+	FIELD field_205 rainGradientPrev F
+	FIELD field_206 rainGradient F
+	FIELD field_207 thunderGradientPrev F
+	FIELD field_208 thunderGradient F
+	FIELD field_209 ticksSinceLightning I
+	FIELD field_210 lightningTicksLeft I
+	FIELD field_211 pauseTicking Z
+	FIELD field_212 saveInterval I
+	FIELD field_213 difficulty I
+	FIELD field_214 random Ljava/util/Random;
+	FIELD field_215 newWorld Z
 	FIELD field_216 dimension Lnet/minecraft/class_50;
+	FIELD field_217 eventListeners Ljava/util/List;
+	FIELD field_218 chunkManager Lnet/minecraft/class_51;
 	FIELD field_219 dimensionData Lnet/minecraft/class_52;
 	FIELD field_220 properties Lnet/minecraft/class_7;
+	FIELD field_221 isEventProcessingEnabled Z
```

Comment:
This is one of the few places with the `isAction` naming convention do we want to switch this not to have `isAction` to match modern yarn boolean naming convention for fields (getters use the `isAction` naming convention instead in modern yarn), so the name would now be `eventProcessingEnabled`

Reply:
will be `eventProcessingEnabled`